www/docs/en/dev/guide/platforms/windows/index.md

```diff
@@ -24,15 +24,15 @@ toc_title: Windows
 # Windows Platform Guide
 
 This guide shows how to set up your SDK development environment to build
-and deploy Cordova apps for Windows 8.1, Windows Phone 8.1, and
-Windows 10 Universal App Platform.  It shows how to use either shell tools
+and deploy Cordova apps Windows 10 (Universal Windows Platform [= UWP], formerly known as Universal App Platform [= UAP]), 
+Windows 8.1 and Windows Phone 8.1.  It shows how to use either shell tools
 to generate and build apps, or the cross-platform Cordova CLI. (See the [Overview](../../overview/index.html#development-paths) for a comparison of these
 development options.) This section also shows how to modify Cordova apps
 within Visual Studio. Regardless of [which](../../overview/index.html#development-paths) approach you take, you need to
 install the Visual Studio SDK, as described below.
 
-Developers wishing to target Windows Phone 8 should use the wp8 platform,
-see [Windows Phone 8 Platform Guide](../wp8/index.html) for details (Warning, the wp8 platform is deprecated).
+Note: Developers wishing to target Windows Phone 8 (instead of 8.1!) should use the `wp8` platform,
```

Comment:
IMO it is not necessary to talk about or suggest to users to use a deprecated platform.

Reply:
As I understand this this is historically as both grew from the same platform. But this change just clarifies this existing sentence - didn't just want to remove it.

Reply:
Oh, that platform does not exist any more at all in the docs - so this indeed does make no sense to mention here this way.

Reply:
Removed.